gammapy/spectrum/__init__.py

```diff
@@ -12,3 +12,4 @@
 from .fit import *
 from .results import *
 from .sensitivity import *
+from .spectrum_dataset import *
```

Comment:
Maybe just name the submodule `.dataset`, because it already lives in the `gammapy.spectrum` sub-package?

Reply:
Done